src/scenes/ContactUsRouter/ContactUsHome.tsx

```diff
@@ -94,10 +95,11 @@ export default function ContactUsHome(): JSX.Element {
     () =>
       (types ?? []).map(
         (item): ListItemContent => ({
-          title: item.name,
-          description: item.description,
-          buttonText: item.name,
-          onClick: () => goToContactUsForm(item.requestTypeId),
+          icon: getSupportRequestIconName(item),
```

Comment:
Since some of these are pulling values from `strings`, you have to add `activeLocale` to the dependencies and check for it